packages/blade/src/components/Dropdown/__tests__/Dropdown.web.test.tsx

```diff
@@ -533,11 +552,29 @@ describe('<Dropdown />', () => {
 describe('<Dropdown /> with <DropdownButton />', () => {
   // Skipping this test because the id that `useId` generates seems to be different and flaky between local and CI.
   // Have to figure out solution to that and then enable this again.
-  it.skip('should render menu and make items clickable', async () => {
```

Comment:
Unrelated change: I broke this test into 2 parts. The snapshot and non-snapshot assertions.

This will allow us to do non-snapshot assertions at least while the snapshots are being skipped because of flaky id generation